sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_options.hpp

```diff
@@ -18,59 +18,23 @@
 
 namespace Azure { namespace Security { namespace KeyVault { namespace Certificates {
 
-  /**
-   * @brief The API version to use from Key Vault.
-   *
-   */
-  class ServiceVersion final {
-  private:
-    std::string m_version;
-
-  public:
-    /**
-     * @brief Construct a new Service Version object
-     *
-     * @param version The string version for the Key Vault Certificate service.
-     */
-    ServiceVersion(std::string version) : m_version(std::move(version)) {}
-
-    /**
-     * @brief Enable comparing the ext enum.
-     *
-     * @param other Another #ServiceVersion to be compared.
-     */
-    bool operator==(ServiceVersion const& other) const { return m_version == other.m_version; }
-
-    /**
-     * @brief Return the #ServiceVersion string representation.
-     *
-     */
-    std::string const& ToString() const { return m_version; }
-
-    /**
-     * @brief Use to send request to the 7.2 version of Key Vault service.
-     *
-     */
-    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const ServiceVersion V7_2;
-  };
-
   /**
    * @brief Define the options to create an SDK Certificate client.
    *
    */
   struct CertificateClientOptions final : public Azure::Core::_internal::ClientOptions
   {
-    ServiceVersion Version;
+    /**
+     * @brief Service Version used.
+     *
+     */
+    std::string Version;
```

Comment:
All the changes around removing `ServiceVersion` and leaving things as `string` seem to be breaking changes. Update the changelog to call that out.

Also, why did we decide to do it this way, rather than using the `ServiceVersion` class?

Reply:
Yes, ask Jeff, we want to make it obvilus for consumers that while you can use different service version we do not guarantee the result, we only guarantee with the latest version :)